gamemodes/terrortown/gamemode/client/cl_keys.lua

```diff
@@ -4,6 +4,7 @@
 
 local timer = timer
 local IsValid = IsValid
+local cv_sv_cheats = GetConVar("sv_cheats")
```

Comment:
Does it already exists there?

Reply:
it does work, and as this is a predefinied ConVar, i suppose it exists at start